pkg/metricprovider/custom_metric_provider.go

```diff
@@ -70,89 +70,95 @@ func (p *CustomMetricProvider) GetMetricByName(ctx context.Context, name types.N
 func (p *CustomMetricProvider) GetMetricBySelector(ctx context.Context, namespace string, selector labels.Selector, info provider.CustomMetricInfo, metricSelector labels.Selector) (*custom_metrics.MetricValueList, error) {
 	klog.Info(fmt.Sprintf("Get metric by selector for custom metric, Info %v namespace %s selector %s metricSelector %s", info, namespace, selector.String(), metricSelector.String()))
 
```

Comment:
Since all the prediction metric should be external metric，so custom metric provide should only leave the logic for remote adapter